app/components/Autopay/AutopayList.js

```diff
@@ -40,4 +30,4 @@ AutopayList.defaultProps = {
   merchants: [],
 }
 
-export default AutopayList
+export default injectIntl(AutopayList)
```

Comment:
Same here. `injectIntl` only needs to be applied if we need to access the `intl` prop.

Reply:
I guess I was moving headers between components and this is a leftover